### DISCUSSION TITLE
Error at parameter 'options': Unexpected property: 'incognito'.

### DISCUSSION THREAD

**Discussion by vadese** [aclap-dev/video-downloadhelper#306] [Help and Q&A]
### Your question

Hello, downloads from this URL have worked without problems for the last few years, but this error has been occurring for a few days now:

Video URL: https://www.zdf.de/serien/die-rosenheim-cops/der-schluessel-zur-loesung-100.html

Error: 
Error in invocation of downloads.download(downloads.DownloadOptions options, optional function callback): Error at parameter 'options': Unexpected property: 'incognito'.
TypeError: Error in invocation of downloads.download(downloads.DownloadOptions options, optional function callback): Error at parameter 'options': Unexpected property: 'incognito'.
    at chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:1:7834
    at new Promise (<anonymous>)
    at Proxy.<anonymous> (chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:1:7515)
    at Object.apply (chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:1:7916)
    at R2 (chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:44:9777)
    at async ri (chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:44:13229)

Greetings, Vadese

### Addon & CoApp version and details

OS Windows 11 Pro
VDH Version 8.2.2.8
VdhCoApp 2.0.19
Browser Edge 122.0.2365.92 (Offizielles Build) (64-Bit)

**Comment by GrampaWildWilly** [comment]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3296, Firefox 123.0.1 64-bit, licensed VDH 8.2.2.8a1 beta, CoApp 2.0.19.  I do not work for VDH.  I am just another user.

I visited your web page & this is what I got.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/347bf0d6-034e-4121-967c-ab3e8ecc253a)

You'll notice everything is in English.  Google so kindly offered to translate the page for me so I let it.  It seems to have done a decent job.  I don't speak German so I'm just trusting it.  In any case, the VDH menu was empty at first.  So I clicked play & paused it after 5 seconds, as you can see in that image.  That made the one variant appear on the VDH menu.  So I downloaded it.  Here's how that looked:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/d62e1003-d114-4308-a139-9014abef30de)
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/4af3c1ac-5e2e-4f6d-b541-a75504968df7)

That's a really good download speed.  I've tried to look at ZDF content in the past & I've been geoblocked, so I feel lucky this even let me in.  To get such great download speed is a bonus.

The download eventually completed & I got this:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/ccc92099-8b79-48e9-a855-a151c4aaeabb)

It played fine in VLC:

![#05](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/387f59e7-9e9c-4633-80ca-b541f6d9f516)

You can see the video was just fine.  You'll have to take my word for it that the audio was just as fine, although it was German so I didn't understand a word of it.  That's one reason I didn't sit & watch this.  Another is that I didn't really want to watch this anyway.  I just sampled it at intervals & I was getting both video & audio at each sample point, including the last few seconds.

So I don't know what to tell you.  You're using Edge, I'm using Firefox.  It wouldn't be the first time something worked better in Firefox that it did on another browser.  All I can tell you is that this worked fine for me.

**Comment by paulrouget** [comment]
I broke that, see my reply here: https://github.com/aclap-dev/video-downloadhelper/discussions/308#discussioncomment-8834832
